lib/__tests__/index-test.js

```diff
@@ -667,14 +689,12 @@ describe('device is set to mobile', () => {
 });
 
 describe('exclusions', () => {
-  var createElement = React.createElement;
-
   before(() => {
     a11y(React, { exclude: ['REDUNDANT_ALT'] });
   });
 
   after(() => {
-    React.createElement = createElement;
+    a11y.restore(React);
```

Comment:
@ljharb I updated the name on `after` module to `updatePatchedMethods`. Naming is hard. PTAL